doc/developer/platform-checks.md

```diff
@@ -47,7 +47,7 @@ Running validate() from <materialize.checks.threshold.Threshold object at 0x7f98
 You can check if the failure is reproducible in isolation by running just the Check in question against just the Scenario in question:
 
 ```
-./mzcompose run default --scenario=RestartEnvironmentdClusterdStorage --check=Threshold
+bin/mzcompose run default --scenario=RestartEnvironmentdClusterdStorage --check=Threshold
```

Comment:
Mzcompose tests are runnable in one of two ways:

```
cd test/path-to-test
./mzcompose run default ...
```

or

```
cd root_of_repository
bin/mzcompose --find path-to-test run default ...
```

The syntax examples previously used No 1, but if you want to convert them to No 2 you need the --find argument to bin/mzcompose

No 1 is easier when developing but No 2 is most useful when git bisecting and in the CI, when the CWD is the root of the repository.